srunner/scenariomanager/atomic_scenario_behavior.py

```diff
@@ -1120,6 +1121,7 @@ def update(self):
         self._actor.set_velocity(carla.Vector3D(0, 0, 0))
         self._actor.set_angular_velocity(carla.Vector3D(0, 0, 0))
         self._actor.set_transform(self._transform)
+        self._actor.set_simulate_physics(enabled=True)
```

Comment:
I tested Control Loss with this change: Due to this change, debris is above the ground and causes ego-vehicle to crash. 
![Screenshot from 2019-04-08 15-47-51](https://user-images.githubusercontent.com/6310385/55717196-6c0bbe00-5a16-11e9-890a-8235cff657f6.png)

Reply:
Good ! I think I noticed it once ! Good catch.

Reply:
Fixed